students/arkadiusz_kasprzyk/lesson_03_functions/badly_organized_calculator.py

```diff
@@ -0,0 +1,79 @@
+# functions
+
+
+def help():
+    """help
+    """
+    print("a - add")
+    print("s - subtract")
+    print("m - multiply")
+    print("d - divide")
+    print("p - power")
+    print("h,? - help")
+    print("q - QUIT")
+
+
+def inputs(message):
```

Comment:
`prompt_for_input` would be a better name and `message` is actually an `operand`, not a whole message.